interface/themes/oe-common/messages-common.scss

```diff
@@ -0,0 +1,29 @@
+/*---------------------------openemr5.0.2 messages--------------------------- */
+.oe-margin-t-3 {
+    margin-top: 3px;
+}
+.oe-margin-t-10 {
+    margin-top: 10px;
+}
+.oe-display {
+    display: none;
+}
+.oe-margin-b-20 {
+    margin-bottom: 20px;
+}
+.oe-margin-b-10 {
+    margin-bottom: 10px;
+}
+a.arrowhead, a:hover.arrowhead, a:visited.arrowhead {
+    color: black;
+}
+.oe-cursor-stop {
+    cursor: not-allowed;
+}
+.oe-cursor-add {
+    cursor: cell;
+}
+.oe-patient-background {
+    background: #ffff9e !important;
+}
+/*---------------------------openemr5.0.2 messages--------------------------- */
```

Comment:
Is this currently being used in the codebase?

Reply:
No, used in my Messages restyled pages :)